### ISSUE TITLE
Human-Readable Parsing Error Message format

### ISSUE THREAD

**Issue by cpotteck** [neoJINXD/Eternity#38]
For an input of `ln(9`, I get the following error:

```
= Parsing Error: Expected {{{{{W:(ABCD...) Suppress:("(")} Group:(Group:(Forward: None) [, Group:(Forward: None)]...)} Suppress:(")")} | {{"E" | "PI"} | Re:('[+-]?\\d+(?:\\.\\d*)?(?:[eE][+-]?\\d+)?')}} | Group:({{Suppress:("(") : ...} Suppress:(")")})}, found end of text (at char 4), (line:1, col:5)
```

The error is expected, but this is not a good format for regular use.

**Comment by EmanuelUnity3D** [comment]
This one is actually fixed already in staging. All parsing errors now output "Parsing Error: Invalid input entered." regardless of what the parsing error is.

**Comment by cpotteck** [comment]
Weird, I thought I created this issue after testing your branch. I'll double check

**Comment by cpotteck** [comment]
@EmanuelUnity3D you're right I must have the timeline mixed up. Closing as fixed by #23
